Registry/Registry.EM_COMMON

```diff
@@ -2566,6 +2569,8 @@ rconfig   integer sf_ocean_physics        namelist,physics      1            0
 rconfig   integer traj_opt                namelist,physics      1            0         h     "traj_opt"    "activate trajectory calculation  0=no, 1=on"   ""
 rconfig   logical dm_has_traj             namelist,physics      max_domains  .false.   rh    "has_traj"    "activate trajectory calculation per domain"   ""
 rconfig   integer tracercall              namelist,physics      1            0         h     "tracercall"  "activate tracer calculation  0=no, 1=on"   ""
+rconfig   integer shalwater_rough         namelist,physics      max_domains  0         rh    "shalwater_rough"     "shallow water roughness flag"      ""
+rconfig   real    shalwater_depth         namelist,physics      1            -1.0      rh    "shalwater_depth"     "water depth for shallow water scheme"      "m"
```

Comment:
@phawbeck The default value set in the registry is -1. What does this signify?

Reply:
Hi @weiwangncar - This would come into play in the following case:
1. User sets shalwater_rough = 1
2. User does not have bathymetry data
3. User did not set shalwater_depth in the namelist
This would cause the value for shalwater_depth to default to -1, and a negative value would cause the model to produce an error. This prevents it from being activated accidentally / erroneously as setting the default value to something like 40 m might be problematic in some situations (e.g., simulations over deep water areas).

That said, I am open to changing it if this is an issue!

Reply:
@phawbeck This is probably ok. Usually the default value put in the registry file is a valid option/value, representing expected default behavior. In this case, it may be ok to force a user to define the depth so that the user knows what is used.